provision-proctor/setup.sh

```diff
@@ -117,6 +97,9 @@ if [[ -z "$proctorNumber" ]]; then
     proctorNumber="$(randomChar;randomChar;randomChar;randomNum;)"
 fi
 
+declare proctorName="monitoring$(randomChar;randomChar;randomChar;randomNum;)"
+declare proctorNumber="$(randomNum;randomNum;randomNum;randomNum;)"
```

Comment:
Lines 95-98 are no longer needed since it will never be passed.

Reply:
Deleted that check.